Resources/Server Info/Guidebook/Science/ScannersAndVessels.xml

```diff
@@ -0,0 +1,24 @@
+<Document>
+## Anomaly Scanner
+
+The Anomaly Scanner is used to gather various information on anomalies. They spawn inside of the lockers in the science department.
+
+<Box>
+<GuideEntityEmbed Entity="AnomalyScanner"/>
+</Box>
+
+Scanning an anomaly shows the severity, general stability, approximate point output, reactions with particle types, and time until the next pulse. The reaction types, specifically, can be used in conjunction with the A.P.E. to manipulate anomalies.
+
+## Anomaly Vessel
+
+The anomaly vessel is a machine that generates points from anomalies. Simply click on an empty vessel with an anomaly scanner, and the anomaly that was last scanned will be synced with the vessel.
```

Comment:
Maybe write "use the anomaly scanner on an empty vessel" instead of "click on an empty vessel with an anomaly scanner".